### ISSUE TITLE
kubelet should ignore files that start with '.'

### ISSUE THREAD

**Issue by thockin** [kubernetes/kubernetes#46171]
Was editing a manifest and found this in the logs:

```
E0520 21:25:51.800333    2483 file_linux.go:113] can't process config file "/etc/kubernetes/manifests/.kube-proxy.manifest.swp": /etc/kubernetes/manifests/.kube-proxy.manifest.swp: read '<binary gunk>
```

We probably also should not log the whole invalid file.

@kubernetes/sig-node-bugs

**Comment by xiangpengzhao** [comment]
FYI. This was fixed in #45111. The related issue is #44450 .
@yujuhong

**Comment by thockin** [comment]
thanks!
